### DISCUSSION TITLE
some improvement on app-shell docs

### DISCUSSION THREAD

**Discussion by jodhi** [mantinedev/mantine#436] [Feedback]
Hi all,
Just saw the docs related with app-shell missing `theme` var
link: [https://mantine.dev/core/app-shell/#responsive-styles](https://mantine.dev/core/app-shell/#responsive-styles)

maybe we should put  this beforehand
```
const theme = useMantineTheme();
```

Thanks

**Comment by rtivital** [comment]
Yep, we are missing that, can you create a PR with a fix?

**Reply by jodhi** [reply]
of course, coming right up

**Reply by jodhi** [reply]
this is the PR [https://github.com/mantinedev/mantine/pull/443](https://github.com/mantinedev/mantine/pull/443)
